projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -131,9 +131,9 @@ export class TuiInputPhoneInternationalComponent
     @HostListener('paste.capture.prevent.stop', ['$event'])
     @HostListener('drop.capture.prevent.stop', ['$event'])
     onPaste(event: ClipboardEvent | DragEvent) {
-        const value = extractValueFromEvent(event);
+        const value = extractValueFromEvent(event).replace('+', '');
         const countryIsoCode = this.countries.find(countryIsoCode =>
-            value.startsWith(this.isoToCountryCode(countryIsoCode)),
+            value.startsWith(this.isoToCountryCode(countryIsoCode).replace('+', '')),
```

Comment:
Hello, what do you think about?

```ts
const value = escapeInvalidValue(extractValueFromEvent(event));

const countryIsoCode = this.countries.find(countryIsoCode =>
  escapeInvalidValue(value.startsWith(this.isoToCountryCode(countryIsoCode))
);
```

```ts
export function escapeInvalidValue(value: string): string {
 return value.replace('+', '');
}
```

`escapeInvalidValue` - not sure about that is good name

Reply:
I think it's ok as is, it's just a few characters in one spot.